Dockerfile

```diff
@@ -30,7 +30,6 @@ RUN rm /usr/local/etc/php-fpm.conf \
     && sed -i s/ssl_session_cache/#ssl_session_cache/g /etc/nginx/nginx.conf \
     && mkdir -p /var/run/php /var/run/nginx
 
-COPY .github/docker/default.conf /etc/nginx/http.d/default.conf
```

Comment:
I'd rather leave this just in case.

Reply:
Leaving it in will cause the if check to fail in entrypoint.sh which determines if the config was already in place or not

Reply:
This one to be specific `if [ -f /etc/nginx/conf.d/default.conf ]; then`
@parkervcp

Reply:
Another way of solving it would be to change the name of the custom conf to be `panel.conf` and have a check on that and remove the default conf when its configured